test/cluster/auth_cluster/test_raft_service_levels.py

```diff
@@ -379,9 +380,11 @@ async def test_workload_prioritization_upgrade(manager: ManagerClient):
     # Using error injection, the test disables WORKLOAD_PRIORITIZATION feature
     # and removes `shares` column from system_distributed.service_levels table.
     config = {
+        **auth_config,
         'authenticator': 'AllowAllAuthenticator',
         'authorizer': 'AllowAllAuthorizer',
         'force_gossip_topology_changes': True,
+        'enable_tablets': False,
```

Comment:
Why did you add enable_tablets=false here?

Reply:
because test want to start  force_gossip_topology_changes': True and in suite.yaml(default settings) enable_tablets=true

with force_gossip_topology_changes': True  and enable_tablets=true node just don't start